tests/ecn/test_red_accuracy.py

```diff
@@ -0,0 +1,101 @@
+import pytest
```

Comment:
Add a skip in the test function along with comments that this test is not intended to be run in nightly

Reply:
Example: https://github.com/Azure/sonic-mgmt/blob/master/tests/qos/args/qos_sai_args.py 'disable_test'
https://github.com/Azure/sonic-mgmt/blob/master/tests/qos/test_qos_sai.py#L289

Reply:
Fixed